include/godot_cpp/variant/callable_method_pointer.hpp

```diff
@@ -31,6 +31,20 @@
 #ifndef GODOT_CALLABLE_METHOD_POINTER_HPP
 #define GODOT_CALLABLE_METHOD_POINTER_HPP
 
+#ifdef GODOT_MODULE_COMPAT
+#include <drivers/gles3/rasterizer_canvas_gles3.h>
+#include <drivers/vulkan/rendering_device_driver_vulkan.h>
+#include <drivers/d3d12/rendering_device_driver_d3d12.h>
+#include <core/object/callable_method_pointer.h>
+#include <core/string/node_path.h>
+#include <scene/3d/node_3d.h>
+#include <scene/gui/control.h>
+#include <scene/main/node.h>
+#include <servers/rendering/renderer_geometry_instance.h>
+#include <servers/rendering/rendering_light_culler.h>
+#include <servers/rendering/renderer_rd/storage_rd/particles_storage.h>
```

Comment:
These were auto generated as well, it's possible some of the auto generated files will have invalid imports too, might need to test some time. Maybe make this commit and then see what works and what not.

Reply:
In any case, will update and check all manual files to make sure they are correct.

Reply:
More I think, more maybe we should indeed have a file that maps everything, just so we can do manual updates to it too in case code I wrote doesn't work well.